src/koheesio/spark/secrets/databricks.py

```diff
@@ -0,0 +1,82 @@
+"""Module for retrieving secrets from DataBricks Scopes.
+
+Secrets are stored as SecretContext and can be accessed accordingly.
+
+See DataBricksSecret for more information.
+"""
+
+from typing import Dict, Optional
+import re
+
+from koheesio.models import Field, model_validator
+from koheesio.secrets import Secret
+from koheesio.spark.utils import on_databricks
+
+
+class DataBricksSecret(Secret):
+    """
+    Retrieve secrets from DataBricks secret scope and wrap them into Context class for easy access.
+    All secrets are stored under the "secret" root and "parent". "Parent" either derived from the
+    secure scope by replacing "/" and "-", or manually provided by the user.
+    Secrets are wrapped into the pydantic.SecretStr.
+
+    Example:
+    ```python
+    context = {"secrets": {"parent": {"webhook": SecretStr("**********"), "description": SecretStr("**********")}}}
+    ```
+
+    Values can be decoded like this:
+    ```python
+    context.secrets.parent.webhook.get_secret_value()
+    ```
+    or if working with dictionary is preferable:
+    ```python
+    for key, value in context.get_all().items():
+        value.get_secret_value()
+    ```
+    """
+
+    scope: str = Field(description="Scope")
+    alias: Optional[Dict[str, str]] = Field(default_factory=dict, description="Alias for secret keys")
+
+    @model_validator(mode="before")
+    def _set_parent_to_scope(cls, values):
+        """
+        Set default value for `parent` parameter on model initialization when it was not
+        explicitly set by the user. In this scenario scope will be used:
+
+        'secret-scope' -> secret_scope
+        """
+        regex = re.compile(r"[/-]")
+        path = values.get("scope")
+
+        if not values.get("parent"):
+            values["parent"] = regex.sub("_", path)
+
+        return values
+
+    @property
+    def _client(self):
+        """
+        Instantiated Databricks client.
+        """
+        if not on_databricks():
+            raise RuntimeError("dbutils not available")
+
+        from pyspark.dbutils import DBUtils  # pylint: disable=E0611,E0401 # type: ignore
+        from pyspark.sql import SparkSession
+
+        dbutils = DBUtils(SparkSession.getActiveSession())
+
+        return dbutils
```

Comment:
Should this be moved to [utils](https://github.com/Nike-Inc/koheesio/blob/main/src/koheesio/utils.py) for example or in `integrations\databricks`? 
I feel like `dbutils` might be useful in some other future functionalities

Reply:
I agree with @maxim-mityutko

Reply:
Moved